### ISSUE TITLE
Documantation problem

### ISSUE THREAD

**Issue by yunusc5n** [Bearle/django_private_chat2#1]
https://django-private-chat2.readthedocs.io/ is giving 404.

**Comment by delneg** [comment]
documentation WILL BE :)

it's not yet...

**Comment by rathgamer** [comment]
Still the problem isn't resolved. Since, I am new to django... idk where to start and things work...

**Comment by yunusc5n** [comment]
This repo is new and the developer/s are trying to write a documentation I think. You should consider using another project or repo for beginning. My suggestion is you should firstly understand the basics of django (routing, apps, settings etc.) with simple website or page and then use a module.

**Comment by rathgamer** [comment]
I do have knowledge about how things work in django wsgi applications. But this is my first time to work with channels, sockets and all those stuffs ...

**Comment by delneg** [comment]
> I do have knowledge about how things work in django wsgi applications. But this is my first time to work with channels, sockets and all those stuffs ...

Please study the example application in the meantime.

**Comment by Flamzyfreeman** [comment]
I want to be able to use my own create login with Django-private-chat, please can someone help put me through.

**Comment by andrewcirel** [comment]
I've added the django-private-chat2 to my project and I can use the admin, get a response from the dialog and message URLs, but am now trying to do the rest.  Not sure where to start so thought I would look at the example app mentioned on the django-private-chat2 webpage, but couldn't see any links?  Am I missing something?
I tried the tox install using my Apple Mac which worked without errors, but when I try to run tox I get the error: ERROR: tox config file (either pyproject.toml, tox.ini, setup.cfg) not found
Is there any example app?
